microsoft-graph.d.ts

```diff
@@ -1,10 +1,11 @@
-// Project: https://github.com/microsoftgraph/msgraph-beta-typescript-typings
+// Project: https://github.com/microsoftgraph/msgraph-typescript-typings
```

Comment:
This change is incorrect.

Reply:
linking original comments: https://github.com/microsoftgraph/msgraph-beta-typescript-typings/pull/39#discussion_r703859073